app/presenters/pagination_presenter.rb

```diff
@@ -0,0 +1,78 @@
+class PaginationPresenter
```

Comment:
DraperDecorator decorates the model objects, however it doesn't seem to do well with collections. Added to that is the fact that this method works with paginated collections.